.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: rust
+rust:
+- stable
+- beta
+- nightly
+sudo: false
+install:
+- wget https://github.com/xiph/speexdsp/releases/download/SpeexDSP-1.2rc3.tar.gz
```

Comment:
Would be better pulling from git in this case

Reply:
you mean cloning the repo? I'm okay with that but a tar might have been faster

Reply:
The release code is pretty old sadly ^^;